src/emulator/shader/src/usse_translator_entry.cpp

```diff
@@ -37,7 +37,7 @@ using USSEMatcher = shader::decoder::Matcher<Visitor, uint64_t>;
 template <typename V>
 boost::optional<const USSEMatcher<V> &> DecodeUSSE(uint64_t instruction) {
     static const std::vector<USSEMatcher<V>> table = {
-    // clang-format off
+// clang-format off
```

Comment:
remove this change

Reply:
Why the heck does clang format keep on changing that >:(